[vtk] Add missing member field initialization

This patch has been upstreamed and will no longer apply, it
can be deleted the next time drake upgrades VTK.

https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10860

--- Filters/Sources/vtkCylinderSource.cxx
+++ Filters/Sources/vtkCylinderSource.cxx
@@ -26,6 +26,7 @@ vtkCylinderSource::vtkCylinderSource(int res)
   this->Radius = 0.5;
   this->Capping = 1;
   this->CapsuleCap = 0;
+  this->LatLongTessellation = 0;
   this->Center[0] = this->Center[1] = this->Center[2] = 0.0;
   this->OutputPointsPrecision = SINGLE_PRECISION;
 
